sentry-kotlin-multiplatform/src/commonAppleMain/kotlin/io/sentry/kotlin/multiplatform/nsexception/Throwable.kt

```diff
@@ -22,9 +22,8 @@ package io.sentry.kotlin.multiplatform.nsexception
 internal val Throwable.causes: List<Throwable> get() = buildList {
     val causes = mutableSetOf<Throwable>()
     var cause = cause
-    while (cause != null && cause !in causes) {
+    while (cause != null && causes.add(cause)) {
```

Comment:
Is this just an optimization? can `causes.add` return false for some reason?

Reply:
yup, just a minor optimization - `add(...)` will return false on elements that are already in a set.